docs/cloud/managing-airbyte-cloud.md

```diff
@@ -274,9 +276,75 @@ Airbyte recommends that you reset streams. A reset will delete data in the desti
 To refresh the source schema:
 1. Click **Refresh source schema** to fetch the schema of your data source.
 
-2. If there are changes to the schema, the **Refreshed source schema** dialog displays them.
+2. If the schema has changed, the **Refreshed source schema** dialog displays them.
+
+## Manage schema changes
+
+Airbyte detects changes in your source schema and allows you to review the changes.
+
```

Comment:
@Amruta-Ranade what do you think about grouping this information in a note here?